README.md

```diff
@@ -1,27 +1,45 @@
 go-skeleton
 -----------
-A skeleton for go microservices
+This project is used as boilerplate example for the structure of a go microservice. A template to get you started so to speak.
+
+A skeleton for go microservices was the core idea, so that's where the name comes from.
 
 
 ## Motivation
 Besides avoiding to copy it from the last microservice I've written, a friend asked how I'd do it.
 That's is how I'm doing it right now
 
 
+## Download
+
+Clone the repository or use 
+```bash
+go get -u github.com/AndersonQ/go-skeleton.git
```

Comment:
please remove the `.git` I guess you added back by mistake
```suggestion
go get -u github.com/AndersonQ/go-skeleton
```

Reply:
sorry about that